### PR TITLE
84056: Update README and change to MarkDown file

### PR DESCRIPTION
**Pull request by balexandr**
Update the README

## Summary

- Change README to MarkDown
- Update with new info

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/84056

### CODE REVIEW

**Comment by cloudmagic80** [comment]
I've added documentation about the uploads_controller.rb

**Review by michaelclement — APPROVED** [review]

**Review by reiting — APPROVED** [review]

**Review by cloudmagic80 — APPROVED** [review]

File: modules/ivc_champva/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+[Swagger API UI](https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/) then search "https://dev-api.va.gov/v1/apidocs" to see the ivc_champva endpoint
+[Project MarkDowns](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/champva)
+[DataDog Dashboard](https://vagov.ddog-gov.com/dashboard/zsa-453-at7/ivc-champva-forms)
+[Pega Callback API ADR](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/champva/ADR-callback-api-to-receive-status-from-pega.md)
+[Pega Callback API Implementation Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/champva/callback-api-technical-spec.md)
+
+## Endpoints
+/ivc_champva/v1/forms
+/ivc_champva/v1/forms/submit_supporting_documents
+/ivc_champva/v1/forms/status_updates
```
Comment (rmtolmach) [inline modules/ivc_champva/README.md:25]: These also show up as one line.

```suggestion
- /ivc_champva/v1/forms
- /ivc_champva/v1/forms/submit_supporting_documents
- /ivc_champva/v1/forms/status_updates
```

or
```suggestion
- `/ivc_champva/v1/forms`
- `/ivc_champva/v1/forms/submit_supporting_documents`
- `/ivc_champva/v1/forms/status_updates`
```

File: modules/ivc_champva/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+### Generate files for new forms
+rails ivc_champva:generate\['path to PDF file'\]
```
Comment (rmtolmach) [inline modules/ivc_champva/README.md:29]:
```suggestion
`rails ivc_champva:generate\['path to PDF file'\]`
```

**Review by rmtolmach — APPROVED** [review]
https://github.com/department-of-veterans-affairs/vets-api/blob/6514d76d8f0948c61cce00b837a7b07ed52e9686/modules/ivc_champva/README.md LGTM ✅

**Pull request merged**